src/arrayMethodSort.js

```diff
@@ -3,9 +3,24 @@
 /**
  * Implement method Sort
  */
+function defaultCompare(a, b) {
+  return String(a) > String(b) ? 1 : -1;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 1; j < this.length - i; j++) {
+        if (compareFunction(this[j - 1], this[j]) > 0) {
+          const item = this[j - 1];
+
+          this[j - 1] = this[j];
+          this[j] = item;
+        }
```

Comment:
The swap operation can be simplified using array destructuring: `[this[j - 1], this[j]] = [this[j], this[j - 1]]`. This removes the need for a temporary variable.